packages/auto-approve/test/auto-approve.test.ts

```diff
@@ -115,6 +122,7 @@ describe('auto-approve', () => {
         checkPRAgainstConfigStub.returns(true);
         validateSchemaStub.returns(undefined);
         checkCodeOwnersStub.returns('');
+        getSecretStub.returns(new Octokit({auth: '123'}));
```

Comment:
Can you add a spy on the Octokit so that we can test that the bot is using this Octokit?